src/mpid/ch4/src/ch4_proc.c

```diff
@@ -11,6 +11,10 @@
  * Refer to comment in ch4_proc.h
  */
 
+#ifndef MPIDI_CH4_AVTABLE_USE_DDR
+#define MPIDI_CH4_AVTABLE_USE_DDR    1
+#endif
```

Comment:
Just define it. We don't want to have the situation (or imply) of hidden defines in multiple places.

Reply:
changed